sdk/go/common/apitype/core.go

```diff
@@ -329,6 +329,10 @@ type ResourceV3 struct {
 	// If set, the providers Delete method will not be called for this resource
 	// if specified resource is being deleted as well.
 	DeletedWith resource.URN `json:"deletedWith,omitempty" yaml:"deletedWith,omitempty"`
+	// Created tracks when this resource was created. Checkpoints prior to early 2023 do not include this.
+	Created *time.Time `json:"created,omitempty" yaml:"created,omitempty"`
+	// Modified tracks the last time this resource's state was changed for any reason except deletions.
```

Comment:
Is this useful? Should this just track Updates?
Maybe we should have both "Updated" to track the last time the engine made a change, "Modified" for the last time the engine did an import/refresh?
```suggestion
	// Modified tracks the last time this resource's state was changed for any reason except deletions. Checkpoints prior to early 2023 do not include this.
```